diff --git a/node_modules/react-native-pip-android/android/build.gradle b/node_modules/react-native-pip-android/android/build.gradle
index d7f0428..509637e 100644
--- a/node_modules/react-native-pip-android/android/build.gradle
+++ b/node_modules/react-native-pip-android/android/build.gradle
@@ -18,11 +18,11 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('PipAndroid_compileSdkVersion', 29)
+    compileSdkVersion safeExtGet('PipAndroid_compileSdkVersion', 34)
     buildToolsVersion safeExtGet('PipAndroid_buildToolsVersion', '29.0.2')
     defaultConfig {
-        minSdkVersion safeExtGet('PipAndroid_minSdkVersion', 16)
-        targetSdkVersion safeExtGet('PipAndroid_targetSdkVersion', 29)
+        minSdkVersion safeExtGet('PipAndroid_minSdkVersion', 24)
+        targetSdkVersion safeExtGet('PipAndroid_targetSdkVersion', 34)
         versionCode 1
         versionName "1.0"
 
diff --git a/node_modules/react-native-pip-android/eas.json b/node_modules/react-native-pip-android/eas.json
new file mode 100644
index 0000000..fead124
--- /dev/null
+++ b/node_modules/react-native-pip-android/eas.json
@@ -0,0 +1,21 @@
+{
+  "cli": {
+    "version": ">= 16.32.0",
+    "appVersionSource": "remote"
+  },
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    },
+    "preview": {
+      "distribution": "internal"
+    },
+    "production": {
+      "autoIncrement": true
+    }
+  },
+  "submit": {
+    "production": {}
+  }
+}
